build.gradle

```diff
@@ -100,107 +78,106 @@ sourceSets {
+    implementation "org.grails.plugins:ala-auth:6.0.0-SNAPSHOT"
+    implementation "org.grails.plugins:ala-ws-security-plugin:6.0.0-SNAPSHOT"
+    implementation "org.grails.plugins:ala-ws-plugin:6.0.0-SNAPSHOT"
+    implementation "org.grails.plugins:audit-logging:4.0.3"
+    implementation 'dk.glasius:external-config:3.1.1'
+    implementation "org.grails.plugins:ala-admin-plugin:2.3.0"
+    implementation 'au.org.ala.plugins:openapi:1.1.0'
 
     console "org.grails:grails-console"
     profile "org.grails.profiles:web"
-    runtime "org.glassfish.web:el-impl:2.1.2-b03"
-    runtime "com.h2database:h2"
-    runtime "org.apache.tomcat:tomcat-jdbc"
-    runtime "javax.xml.bind:jaxb-api:2.3.1"
-    runtime "com.bertramlabs.plugins:asset-pipeline-grails:3.2.4"
-    testCompile "io.micronaut:micronaut-inject-groovy"
-    testCompile "org.grails:grails-gorm-testing-support"
-    testCompile "org.mockito:mockito-core"
-    testCompile "org.grails:grails-web-testing-support"
-    testCompile "org.grails.plugins:geb"
-    testCompile "org.seleniumhq.selenium:selenium-remote-driver:3.141.59"
-    testCompile "org.seleniumhq.selenium:selenium-api:3.141.59"
-    testCompile "org.seleniumhq.selenium:selenium-support:3.141.59"
-    testRuntime "org.seleniumhq.selenium:selenium-chrome-driver:3.141.59"
-    testRuntime "org.seleniumhq.selenium:selenium-firefox-driver:3.141.59"
-    compile 'org.grails.plugins:sentry:11.7.25'
-    // In newer versions use 'implementation'
-    compile 'org.liquibase:liquibase-core:3.10.3'
-    compile "org.grails.plugins:database-migration:3.1.0"
+    runtimeOnly "org.glassfish.web:el-impl:2.1.2-b03"
+    runtimeOnly "com.h2database:h2"
+    runtimeOnly "org.apache.tomcat:tomcat-jdbc"
+    runtimeOnly "javax.xml.bind:jaxb-api:2.3.1"
+    runtimeOnly "com.bertramlabs.plugins:asset-pipeline-grails:3.4.6"
+    testImplementation "io.micronaut:micronaut-inject-groovy"
+    testImplementation "org.grails:grails-gorm-testing-support"
+    testImplementation "org.mockito:mockito-core"
+    testImplementation "org.grails:grails-web-testing-support"
+    testImplementation "org.grails.plugins:geb"
+    testImplementation "org.seleniumhq.selenium:selenium-remote-driver:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-api:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-support:$seleniumVersion"
+    runtimeOnly "net.sourceforge.htmlunit:htmlunit:2.18"
+    testImplementation "org.seleniumhq.selenium:selenium-htmlunit-driver:2.47.1"
+    testImplementation "org.seleniumhq.selenium:selenium-chrome-driver:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-firefox-driver:$seleniumVersion"
+    testImplementation "org.seleniumhq.selenium:selenium-safari-driver:$seleniumSafariDriverVersion"
+    implementation 'org.grails.plugins:sentry:11.7.25'
+    // db-migration
+    implementation 'org.liquibase:liquibase-core:4.19.0'
```

Comment:
Thinking in the grails migration, I was worried about this liquibase part, bit I see you did it great. But in spatial-service for some reason the liquibase support was removed https://github.com/AtlasOfLivingAustralia/spatial-service/issues/213

Reply:
@adam-collins Re: spatial liquibase.